packages/pluggableWidgets/maps-native/src/Maps.tsx

```diff
@@ -200,17 +198,19 @@ export class Maps extends Component<Props, State> {
 
     private async getCenter(): Promise<LatLng> {
         const center =
-            this.props.markers.length === 1 && this.props.fitToMarkers
+            (this.props.centerLatitude && this.props.centerLongitude) || this.props.centerAddress
+                ? await this.parseCoordinate(
+                      this.props.centerLatitude,
+                      this.props.centerLongitude,
+                      this.props.centerAddress
+                  )
+                : this.props.markers.length === 1 && this.props.fitToMarkers
                 ? await this.parseCoordinate(
                       this.props.markers[0].latitude,
                       this.props.markers[0].longitude,
                       this.props.markers[0].address
                   )
-                : await this.parseCoordinate(
-                      this.props.centerLatitude,
-                      this.props.centerLongitude,
-                      this.props.centerAddress
-                  );
+                : null;
```

Comment:
Why dont we return `{ latitude: 51.9066346, longitude: 4.4861703 }` instead of null. Then you can return just`center`

Reply:
done